arm: right
controlled_joints:
- odom_x_joint
- odom_y_joint
- odom_z_joint
- triangle_base_joint
- right_arm_0_joint
- right_arm_1_joint
- right_arm_2_joint
- right_arm_3_joint
- right_arm_4_joint
- right_arm_5_joint
- right_arm_6_joint
goal_pose_name: cup_gp3
projection_mode: false
sim_frequency: 100
simulated_joints:
- odom_x_joint
- odom_y_joint
- odom_z_joint
- left_arm_0_joint
- left_arm_1_joint
- left_arm_2_joint
- left_arm_3_joint
- left_arm_4_joint
- left_arm_5_joint
- left_arm_6_joint
- right_arm_0_joint
- right_arm_1_joint
- right_arm_2_joint
- right_arm_3_joint
- right_arm_4_joint
- right_arm_5_joint
- right_arm_6_joint
- triangle_base_joint
- neck_shoulder_pan_joint
- neck_shoulder_lift_joint
- neck_elbow_joint
- neck_wrist_1_joint
- neck_wrist_2_joint
- neck_wrist_3_joint
- left_gripper_joint
- right_gripper_joint
simulated_links:
- base_link
- triangle_base_link
- triangle_right_arm_link
- calib_right_arm_base_link
- right_arm_1_link
- right_arm_2_link
- right_arm_3_link
- right_arm_4_link
- right_arm_5_link
- right_arm_6_link
- right_arm_7_link
- right_arm_flange_link
- right_arm_cabling_adaptor_link
- right_arm_adapter_iso50_kms40_frame_in
- right_arm_adapter_iso50_kms40_frame_out
- right_arm_kms40_frame_in
- right_arm_kms40_frame_out
- right_arm_adapter_kms40_fwk050_frame_in
- right_arm_adapter_kms40_fwk050_frame_out
- right_arm_fwk_fwa_050_frame_in
- right_arm_fwk_fwa_050_frame_out
- right_arm_adapter_fwa050_wsg50_frame_in
- right_arm_adapter_fwa050_wsg50_frame_out
- right_gripper_base_link
- right_gripper_tool_frame
start_config:
  left_arm_0_joint: -1.5574374644003413
  left_arm_1_joint: 0.8407944290889994
  left_arm_2_joint: 0.09969375661946397
  left_arm_3_joint: -1.4160946657762408
  left_arm_4_joint: -2.2509232785729343
  left_arm_5_joint: -1.482324185767737
  left_arm_6_joint: -2.324271888716723
  neck_elbow_joint: -0.565377
  neck_shoulder_lift_joint: -3.0928
  neck_shoulder_pan_joint: -1.67144
  neck_wrist_1_joint: 1.23036
  neck_wrist_2_joint: 1.54493
  neck_wrist_3_joint: 0.04506
  odom_x_joint: -2.5
  odom_y_joint: 1.86
  odom_z_joint: -0.07
  right_arm_0_joint: 1.035431542036267
  right_arm_1_joint: -0.6214144842664262
  right_arm_2_joint: -2.5601793369348207
  right_arm_3_joint: -1.4676542755818898
  right_arm_4_joint: -1.2967314329570683
  right_arm_5_joint: 1.85759378992942
  right_arm_6_joint: -1.0270237084541678
  triangle_base_joint: -0.31
watchdog_period: 0.1
